ui/cap-react/src/antd/drafts/components/SideBar/DraftSideBar.js

```diff
@@ -20,15 +20,15 @@ const DraftSideBar = ({ visibleFileDrawer, onClose }) => {
       <SideBar />
     </Drawer>
   ) : (
-    <Col xxl={6}>
+    <Col xxl={6} style={{ height: "100%", overflowY: "scroll" }}>
```

Comment:
why we change here?

Reply:
To make the draft sidebar have its own scrollbar and avoid scrolling the entire page when the form itself is not overflowing